flexport/latitude
# LDS-163 Create Segmented Control Component

## Description:
Implements SegmentedControl component documented in [here](https://flexport.atlassian.net/browse/LDS-163). It is a linear set of buttons where only one can be selected at a time.

## Sandbox server:
https://www-46.sandbox.eng-dev.flexport.com/

## Jira:
[LDS-163](https://flexport.atlassian.net/browse/LDS-163)

## Manual Testing Plan:
1. Run storybook.
2. See SegmentedControl under the 'buttons' section.
3. Confirm SegmentedControl works as expected with knobs properly working.
4. Visit the SegmentedControl docs. (https://www-46.sandbox.eng-dev.flexport.com/design/components/SegmentedControl)
5. Confirm documentation is accurate.
6. Confirm live demo works correctly. 

## Screenshots:

<img width="439" alt="Screen Shot 2019-11-25 at 1 41 16 PM" src="https://user-images.githubusercontent.com/7979583/69581404-4e7e6b00-0f8b-11ea-9b45-19d8a92f9a0f.png">

<img width="409" alt="Screen Shot 2019-11-25 at 1 41 22 PM" src="https://user-images.githubusercontent.com/7979583/69581396-4aeae400-0f8b-11ea-9d6c-4c61e104684c.png">

<img width="745" alt="Screen Shot 2019-11-25 at 1 40 55 PM" src="https://user-images.githubusercontent.com/7979583/69581411-5211f200-0f8b-11ea-8d4a-cf6966f9fbee.png">

![Kapture 2019-11-25 at 13 40 37](https://user-images.githubusercontent.com/7979583/69581952-5985cb00-0f8c-11ea-9077-f480b62b4ca7.gif)
